apm/otel_monitor.go

```diff
@@ -136,7 +141,13 @@ func (m *monitor) Started(ctx context.Context, evt *event.CommandStartedEvent) {
 	}
 	if !m.cfg.CommandAttributeDisabled {
 		if stmt := m.cfg.CommandTransformerFunc(evt.Command); stmt != "" {
-			attrs = append(attrs, semconv.DBStatement(stmt))
+			if formattedStmt, err := extractStatement(evt.CommandName, stmt, false); err == nil && formattedStmt != "" {
+				attrs = append(attrs, semconv.DBStatement(formattedStmt))
+			}
+
```

Comment:
Could we expand with comments here to describe why the stripped attribute  is helpful / what it is? I worry we won't really understand this in the future without having to dig into the PR.

Reply:
done